src/components/AdContainer/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import settings from 'settings';
+import getClassNames from 'helpers/getClassNames';
+import styles from './ad-container.scss';
+
+const AdContainer = ({currentCity = {}, type}) => (
+  <div
+    id={`ad-slot--${type}`}
+    className={getClassNames(styles, ['root', type])}
```

Comment:
As Richard says, just do `className={`ad-slot ad-slot--${type}`}` and be done with it :)